util/sqlgen/default.go

```diff
@@ -63,6 +63,11 @@ const (
 	defaultDeleteLayout = `
 		DELETE
 			FROM {{.Table}}
+			
+			{{if .Using}}
+				USING "{{.Using}}"
```

Comment:
Instead of using quotes directly you should be using another kind of fragment that takes a string and adds the quotes when compiling, see the examples that use `{{.Table}}` or `{{.Columns}}` for instance, like this one:

https://github.com/evalvarez12/db/blob/issue-90/util/sqlgen/default.go#L40